### PR TITLE
Accounting code/react table

### PR DESCRIPTION
**Pull request by Pradip-Singh-X**
added react table, contexts , types, services etc. in accounting code section to fetch accounting codes and render using mui react table v2.

Login Credentials: 
username: "emilys" , 
password: "emilyspass"

image

### CODE REVIEW

File: src/components/LoadingSpinner.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { Stack, CircularProgress } from "@mui/material";
+
+const LoadingSpinner = () => {
+  return (
+    <Stack
+      sx={{
+        height: "60vh",
```
Comment (sudip-sth) [inline src/components/LoadingSpinner.tsx]: Put syyles on seperate style.tsx file

File: src/context/AccountingContext.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+);
+
+export const AccountingProvider: React.FC<{ children: React.ReactNode }> = ({
+  children,
+}) => {
+  const [data, setData] = useState<AccountingCode[]>([]);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    const fetchData = async () => {
```
Comment (sudip-sth) [inline src/context/AccountingContext.tsx]: Dont fetch api in context and use react query

**Review by sudip-sth — CHANGES_REQUESTED** [review]

**Comment by sudip-sth** [comment]
please use other names in your practice application. Lets not use the names we have in our main app like acc codes. and others. And convey this message to the team as well

**Comment by Pradip-Singh-X** [comment]
theme and all the nav item names that matches Varicon main app has changed.

image

**Pull request merged**